contracts/zksync/test/claim_payment.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+. contracts/utils/colors.sh #for ANSI colors
+
+printf "${GREEN}\n=> [ETH] Making Claim Payment${COLOR_RESET}\n"
+
+MM_INITIAL_BALANCE_L1=$(cast balance --rpc-url $ETH_RPC_URL --ether $MM_ETHEREUM_PUBLIC_ADDRESS)
+echo "Initial MM balance L1:"
+echo "$MM_INITIAL_BALANCE_L1"
+
+echo "Initial MM balance L2:"
+npx zksync-cli wallet balance --chain "dockerized-node" --address "$MM_ZKSYNC_WALLET" | grep -E -o "\d+(\.\d+)? ETH"
+
+echo "Initial Escrow balance:"
+npx zksync-cli wallet balance --chain "dockerized-node" --address "$ZKSYNC_ESCROW_CONTRACT_ADDRESS" | grep -E -o "\d+(\.\d+)? ETH"
+
+
+echo "Withdrawing $BRIDGE_AMOUNT_ETH ETH"
+echo "Withdrawing $BRIDGE_AMOUNT_WEI WEI"
+
+cast send --rpc-url $ETH_RPC_URL --private-key $ETH_PRIVATE_KEY \
+  $PAYMENT_REGISTRY_PROXY_ADDRESS "claimPaymentZKSync(uint256, uint256, uint256, uint256, uint256)" \
+  "0" "$USER_ETHEREUM_PUBLIC_ADDRESS_UINT" "$BRIDGE_AMOUNT_WEI" "2000000000" "800"\
+  --value 5000000000000000000
+
+#ele pe eme
+#me revertea con info: None
+#no estoy seguro si existe un diamond proxy en la dada address.
+
+
+
+sleep 15
+
+
+MM_INITIAL_BALANCE_L1=$(cast balance --rpc-url $ETH_RPC_URL --ether $MM_ETHEREUM_PUBLIC_ADDRESS)
+echo "After MM balance L1:"
+echo "$MM_INITIAL_BALANCE_L1"
+
+echo "After MM balance L2:"
+npx zksync-cli wallet balance --chain "dockerized-node" --address "$MM_ZKSYNC_WALLET" | grep -E -o "\d+(\.\d+)? ETH"
+
+echo "After Escrow balance:"
+npx zksync-cli wallet balance --chain "dockerized-node" --address "$ZKSYNC_ESCROW_CONTRACT_ADDRESS" | grep -E -o "\d+(\.\d+)? ETH"
+
+
+
+# starkli call $ESCROW_CONTRACT_ADDRESS get_order_pending u256:0
```

Comment:
remove things from starknet in zksync files

Reply:
This is part of WIP integration test.
The SN comments are taken from SN integration test, and they serve as a guide so that ZKSync integration test replicates SN integration test's behaviour.

If I remove this comment, the integration test will loose it's guide for future developing. If I translate this comment to ZKSync functionality, it is continuing the development of the integration test, which must be addressed in another issue and PR as discussed.